base_wamas/views/wamas_document_default_field_views.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="wamas_document_default_field_view_tree" model="ir.ui.view">
+        <field name="name">wamas.document.default.field.view.tree</field>
+        <field name="model">wamas.document.default.field</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name"/>
+                <field name="code"/>
+                <field name="ttype"/>
+                <field name="length"/>
+                <field name="decimal_place"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="wamas_document_default_field_view_form" model="ir.ui.view">
+        <field name="name">wamas.document.default.field.view.form</field>
+        <field name="model">wamas.document.default.field</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group string="General Information">
+                        <group>
+                            <field name="name"/>
+                            <field name="code"/>
+                        </group>
+                        <group>
+                            <field name="ttype"/>
+                            <field name="length"/>
+                            <field name="decimal_place"/>
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="wamas_document_default_field_view_search" model="ir.ui.view">
+        <field name="name">wamas.document.default.field.view.search</field>
+        <field name="model">wamas.document.default.field</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name"/>
+                <field name="code"/>
+                <field name="ttype"/>
+                <field name="length"/>
+                <field name="decimal_place"/>
+                <separator/>
+                <group expand="1" string="Group By">
+                    <filter name="gb_ttype" string="Type" context="{'group_by':'ttype'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="action_wamas_document_default_field" model="ir.actions.act_window">
+        <field name="name">WAMAS Document Default Field</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">wamas.document.default.field</field>
+        <field name="view_mode">tree,form</field>
+        <field name="search_view_id" ref="wamas_document_default_field_view_search"/>
+        <field name="context">{'gb_element_id': 1}</field>
```

Comment:
@tuantrantg did you mean search default by `gb_ttype`?

Reply:
hi a @dzungtran89 , it's my bad, and I don't think there is any **group by** in here. Because under my understanding about the attached excel file in the ticket, there are only 5 default fields. It's a small amount to use **group by**
--> I'll remove this group by